src/commons/theme.ts

```diff
@@ -1,23 +1,24 @@
 import Joi from 'joi';
 
-const colorSchema = Joi.object().keys({
-  primary: Joi.string().required().example('#ff00ff'),
-  secondary: Joi.string().required().example('#ff00ff'),
-  accent: Joi.string().required().example('#ff00ff'),
-  error: Joi.string().required().example('#ff00ff'),
-  warning: Joi.string().required().example('#ff00ff'),
-  info: Joi.string().required().example('#ff00ff'),
-  success: Joi.string().required().example('#ff00ff'),
-  neutral: Joi.string().required().example('#ff00ff'),
+const colorSchema = Joi.string().required().example('#ff00ff').regex(/^#[0-9a-f]{6}$/i);
```

Comment:
I fixed the length at 6digit hex because on the client side, there is come computing on the hex values to generate the highlight colors for the dots in the map (when editing a line)

https://github.com/withthegrid/platform-client/blob/6d990dd56ab51211d4154a7bbf5835542bfdb96a/src/services/theme/index.ts#L63-L79